applications/trilinos_application/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -778,7 +780,8 @@ class TrilinosBlockBuilderAndSolver
                 if(num_active_indices != 0)
                 {
                     int ierr = Agraph.InsertGlobalIndices(num_active_indices,temp,num_active_indices, temp);
-                    if(ierr < 0) KRATOS_THROW_ERROR(std::logic_error,"Epetra failure found in Agraph.InsertGlobalIndices --> ln 942","");
+                    if(ierr < 0)
+                        KRATOS_ERROR << "Epetra failure found in Agraph.InsertGlobalIndices --> ln 942";
```

Comment:
Note that the line number is already off (at least it looks like it from here)... May I suggest the `__LINE__` preprocessor macro? (or something similar, I don't remember the exact name)